scripts/distribution/macOS-package/NodeConfigurationInstallerPlugin/NodeConfigurationInstallerPlugin/MyInstallerPane.m

```diff
@@ -102,16 +108,11 @@ - (void) saveConfigurationToDisk {
 
 // Checks that both the mainnet and testnet node names:
 // - have length > 0
-// - contain no double quotes
 - (BOOL) nodeNamesAreValid
 {
     NSString *mainnetName = [_oMainnetNodeName stringValue];
     NSString *testnetName = [_oTestnetNodeName stringValue];
-    
-    return      [mainnetName length] > 0
-            &&  [testnetName length] > 0
-            && ![mainnetName containsString:@"\""]
-            && ![testnetName containsString:@"\""];
+    return [mainnetName length] > 0 && [testnetName length] > 0;
```

Comment:
Should there be an upper bound?

Reply:
Can the _oTestnetNodeName and _oMainnetNodeName be NULL?

Reply:
Not directly related to this check, however should it be possible to `toggle` which type of nodes, that should be installed/enabled via the installer ?

Reply:
Added upper bound 👍